packages/amplify_authenticator/lib/src/blocs/auth/auth_bloc.dart

```diff
@@ -152,7 +152,7 @@ class StateMachineBloc {
   ) async* {
     try {
       var result = await _authService.confirmSignIn(
-        code: data.code,
+        confirmationValue: data.confirmationValue,
```

Comment:
This can be a code (for MFA) or a new password (for force password reset). confirmationValue is consistent with the name that is used on the public API for `confirmSignIn()`